types/hapi/test/server/connections.ts

```diff
@@ -11,3 +11,9 @@ server.connection({ port: 8080, labels: 'b' });
 const a = server.select('a');
 
 // a.connections.length === 1
+
+server.connections[0].table().map(({method, path, settings: {description}}) => ({
+    method,
+    path,
+    description
+}));
```

Comment:
You can probably just replace this with something like:

    server.connections[0].table().map(({method, path, settings: {description}}) => ();

Bit shorter.